plugins/Transitions/lang/en.json

```
@@ -1,42 +1,42 @@
 {
     "Transitions": {
         "BouncesInline": "%s bounces",
-        "DirectEntries": "Direct Entries",
+        "DirectEntries": "Direct entries",
         "Transitions": "Transitions",
         "ErrorBack": "Go back to the previous action",
         "ExitsInline": "%s exits",
         "NumPageviews": "%s pageviews",
         "NumDownloads": "%s downloads",
         "NumOutlinks": "%s outlinks",
         "TopX": "Top %s labels",
-        "FeatureDescription": "Transitions gives you a report that shows the things your visitors did directly before and after viewing a certain page. This page will explain how to access, understand, and use the powerful Transitions report.",
+        "FeatureDescription": "Transitions is a report showing the things your visitors did directly before and after viewing a given page. This page explains how to access, understand, and use the powerful \"Transitions\" report.",
         "AvailableInOtherReports": "Did you know? Transitions are also available as a row action in the following reports:",
-        "AvailableInOtherReports2": "Simply hover a row in any of these reports and click on the transition icon %s to launch it.",
-        "FromCampaigns": "From Campaigns",
-        "FromPreviousPages": "From Internal Pages",
+        "AvailableInOtherReports2": "Simply hover a row in any of these reports and click on the transition icon \"%s\" to launch it.",
```

```suggestion
        "AvailableInOtherReports2": "Launch a row in any of these reports by clicking on its transition icon (%s).",
```
@sgiehl Would this work?

---

For the icon, yes. But the other text might sound a bit misleading. You don't launch the row, but it launches the transitions feature. So I would suggest something like:

```suggestion
        "AvailableInOtherReports2": "Simply hover a row in any of these reports and launch transitions by clicking on its icon (%s).",
```